### DISCUSSION TITLE
Ajax with select2

### DISCUSSION THREAD

**Discussion by HazemMohamed252** [statamic/cms#6723] [Q&A] [answered]
How can i get data using Ajax jquery for select2 onchange, example i have empty select2 and when o type minimum 3 characters data retrieved as response and show resullt as dropdown. Please advice. Thanks

**Comment by joshuablum** [comment, marked as answer]
Hey @HazemMohamed252,

I saw that your question got already answered on Discord.
Leaving the same link here: https://makitweb.com/how-to-load-data-using-jquery-ajax-in-select2-laravel-8/
